### ISSUE TITLE
type error at torch.nn.functional.normalize (line 312 modelling.py)

### ISSUE THREAD

**Issue by tomad02** [huggingface/setfit#410]
It doesn't work with a dict 
must be done something like:
sentence_embedding = torch.nn.functional.normalize(outputs['sentence_embedding'], p=2, dim=1)                        
logits,_ = self.model_head(sentence_embedding)

**Comment by tomaarsen** [comment]
Thanks for reporting! Apologies for getting to this so slowly. #450 will solve this in preparation for the v1.0.0 release, which will be merged via #439 soon.

- Tom Aarsen
